executor/modes/local/runner.py

```
@@ -0,0 +1,425 @@
+            result: Optional[Dict[str, Any]] = None,
+        ) -> None:
+            """Report progress via WebSocket instead of HTTP callback."""
+            # Create coroutine and schedule it
+            asyncio.create_task(
+                progress_reporter.report_progress(
+                    progress=progress,
+                    status=status or "",
+                    message=message or "",
+                    result=result,
+                )
+            )
```

Ruff (0.14.14)

Store a reference to the return value of `asyncio.create_task`

(RUF006)